docs/api/browser-window-proxy.md

```diff
@@ -50,4 +50,4 @@ The `BrowserWindowProxy` object has the following instance properties:
 
 #### `win.closed`
 
-A Boolean that is set to true after the child window gets closed.
+A `Boolean` that is set to true after the child window gets closed.
```

Comment:
>  I can just take "the second word" too

Yep. That's what we're currently doing too. https://github.com/electron/electron-docs-linter/blob/3c0128cafa3efecdaecefd094a11c830c096acb4/lib/property.js#L7

It would be great to consolidate our efforts here. Aside from https://github.com/electron/electron/pull/8630, is there some info that you need to extract from the docs that our structured docs are not already providing?